Jenkinsfile

```diff
@@ -24,6 +24,16 @@ if (needSplittingFromWorkspace) {
 } else {
   splits = splitTests count(10)
 }
+
+branches['CI'] = {
+  stage('CI') {
+    node('maven-11') {
+      checkout scm 
+      sh "mvn verify --no-transfer-progress -DskipTests -P jenkins-release"
```

Comment:
I know this does run the javadoc  - but should this prefer to use the incrementals profile give we are using CD (and hence use the incremental profile)?

https://github.com/jenkinsci/pom/blob/jenkins-1.85/pom.xml#L931-L971?

most likely something for a different ticket - I'm not entirely sure how incremental publishing works from custom pipelines.

Reply:
I think this is fine although that could work too